clients/erigon/Dockerfile

```
@@ -1,29 +1,31 @@
+## Build Erigon Via Pre-Built Image:
+ARG user=thorax
+ARG repo=erigon
 ARG branch=devel
-FROM thorax/erigon:$branch
+FROM $user/$repo:$branch
 
 # The upstream erigon container uses a non-root user, but we need
 # to install additional commands, so switch back to root.
 USER root
 
-# Install script tools.
-RUN apk add --update bash curl jq
+RUN apk add --no-cache \
+    bash \
+    curl \
+    jq
```

```suggestion
RUN apk add --no-cache bash curl jq
```

> The --no-cache option allows to not cache the index locally, which is useful for keeping containers small.
> Literally it equals apk update in the beginning and rm -rf /var/cache/apk/* in the end.

Nice, today I learned. Prefer one line though

---

Made some small changes! 

This led down the path of trying to reduce the size of the builder stage only to realize it was redundant for a multi stage docker build so I also learned something :D